### PR TITLE
R4 CarePlan: support _revinclude provenance for CarePlan

### PR DESCRIPTION
**Pull request by khansasultana**
Description
----
updating fhir for careplan revinclude

PR Checklist
----
- [x] Screenshot(s) of changes attached before changes merged.
- [ ] Screenshot(s) of changes attached after changes merged and published.

Screenshot 2021-11-18 at 4 44 24 PM
Screenshot 2021-11-18 at 4 44 41 PM
Screenshot 2021-11-18 at 4 44 57 PM

### CODE REVIEW

File: lib/resources/example_json/r4_examples_care_plan.rb [hunk 1]
```diff
@@ -45,6 +45,121 @@ module Resources
...
     }.freeze
 
+    R4_CAREPLAN_REVINCLUDE_BUNDLE ||= {
+      'resourceType': 'Bundle',
+      'id': '627a14f6-0eda-42ab-ad21-e860f5780a20',
+      'type': 'searchset',
+      'link': [
+        {
+          'relation': 'self',
+          'url': 'http://localhost:3000/r4/2c400054-42d8-4e74-87b7-80b5bd5fde9f/CarePlan?_id=178866310&_revinclude=Provenance%3Atarget'
```
Comment (aki-mathur) [inline lib/resources/example_json/r4_examples_care_plan.rb:55]: Update this url to C1941 domain

File: lib/resources/example_json/r4_examples_care_plan.rb [hunk 2]
```diff
@@ -45,6 +45,121 @@ module Resources
...
+        }
+      ],
+      'entry': [
+        {
+          'fullUrl': 'http://localhost:3000/r4/2c400054-42d8-4e74-87b7-80b5bd5fde9f/CarePlan/178866310',
```
Comment (aki-mathur) [inline lib/resources/example_json/r4_examples_care_plan.rb:60]: Update the URL here aswell

File: lib/resources/example_json/r4_examples_care_plan.rb [hunk 3]
```diff
@@ -45,6 +45,121 @@ module Resources
...
+            'period': {
+              'start': '2020-02-03T19:39:33.000Z'
+            }
+          },
+          'search': {
+            'mode': 'match'
+          }
+        },
+        {
+          'fullUrl': 'http://localhost:3000/r4/2c400054-42d8-4e74-87b7-80b5bd5fde9f/Provenance/INT.doc-178866310',
```
Comment (aki-mathur) [inline lib/resources/example_json/r4_examples_care_plan.rb:105]: Update the url

**Review by aki-mathur — CHANGES_REQUESTED** [review]

File: lib/resources/example_json/r4_examples_care_plan.rb (hunk 1 above)
Comment (khansasultana) [reply lib/resources/example_json/r4_examples_care_plan.rb]: updated

File: lib/resources/example_json/r4_examples_care_plan.rb (hunk 2 above)
Comment (khansasultana) [reply lib/resources/example_json/r4_examples_care_plan.rb]: updated

File: lib/resources/example_json/r4_examples_care_plan.rb (hunk 3 above)
Comment (khansasultana) [reply lib/resources/example_json/r4_examples_care_plan.rb]: updated

File: content/millennium/r4/clinical/care-provision/care-plan.md [hunk 4]
```diff
@@ -55,6 +55,7 @@ _Implementation Notes_
  `patient`    | See notes | [`reference`] | Who care plan is for. Example: `patient=12345`
  `category`   | See notes | [`token`]     | The scope of care plan being searched for. Examples: `category=assess-plan`
  [`_count`]   | N         | [`number`]    | Number of results per page.
+ `_revinclude`| No        | [`token`]  | Provenance resource entries to be returned as part of the bundle. Example:_revinclude=Provenance:target
```
Comment (aki-mathur) [inline content/millennium/r4/clinical/care-provision/care-plan.md:58]: nit: This should be aligned correctly

**Review by aki-mathur — CHANGES_REQUESTED** [review]

File: content/millennium/r4/clinical/care-provision/care-plan.md (hunk 4 above)
Comment (vinuthnag) [reply content/millennium/r4/clinical/care-provision/care-plan.md:58]: +1

**Review by vinuthnag — CHANGES_REQUESTED** [review]

File: content/millennium/r4/clinical/care-provision/care-plan.md (hunk 4 above)
Comment (khansasultana) [reply content/millennium/r4/clinical/care-provision/care-plan.md]: addressed

**Review by aki-mathur — APPROVED** [review]

**Review by vinuthnag — APPROVED** [review]

**Comment by aki-mathur** [comment]
Screenshot 2021-12-02 at 6 25 20 PM

**Pull request merged**